### DISCUSSION TITLE
'number'+'.'  in markdown is not visible in page

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [facebook/docusaurus#7895] [Q&A]
eg: in markdown
![image](https://user-images.githubusercontent.com/15354455/182759463-40b85260-70aa-4754-959c-795bb22d0a4b.png)
but in page the "1." is disappeared！it changed to tag "ol",may be version caused.
![image](https://user-images.githubusercontent.com/15354455/182759154-d40d17ec-5594-46f4-b3e3-551a92175107.png)

my dependencies：
"dependencies": {
    "@docusaurus-plugin/less": "^0.1.1",
    "@docusaurus/core": "^2.0.0-beta.14",
    "@docusaurus/plugin-google-gtag": "^2.0.0-beta.14",
    "@docusaurus/preset-classic": "^2.0.0-beta.14",
    "@mdx-js/react": "^1.6.21",
    "@svgr/webpack": "^5.5.0",
    "ahooks": "^3.1.7",
    "antd": "^4.18.3",
    "aos": "^3.0.0-beta.6",
    "axios": "^0.24.0",
    "clsx": "^1.1.1",
    "docusaurus-plugin-sass": "^0.2.1",
    "file-loader": "^6.2.0",
    "hover.css": "^2.3.2",
    "http-server": "^14.0.0",
    "less": "^4.1.2",
    "less-loader": "^10.2.0",
    "prism-react-renderer": "^1.2.1",
    "rc-texty": "^0.2.0",
    "react": "^17.0.1",
    "react-dom": "^17.0.1",
    "react-svg": "^14.1.15",
    "react-use": "^17.3.2",
    "sass": "^1.43.2",
    "sass-resources-loader": "^2.2.4",
    "serve": "^12.0.1",
    "server": "^1.0.36",
    "swiper": "^7.2.0",
    "url-loader": "^4.1.1"
  },
  "devDependencies": {
    "@docusaurus/module-type-aliases": "^2.0.0-beta.14",
    "@tsconfig/docusaurus": "^1.0.4",
    "postcss-loader": "^6.2.0",
    "postcss-px-to-viewport": "^1.1.1",
    "typescript": "^4.3.5"
  },

**Comment by Josh-Cena** [comment]
I can't tell without seeing more context. The dependency list doesn't necessarily mean what actually gets used on this page. Could you make a minimal reproduction, or share your code?

**Comment by Chasen-Zhang** [comment]
no more codes,  just used markdown.

**Reply by Josh-Cena** [reply]
Yes, I would need a reproduction somehow. Apparently that isn't the behavior in a new Docusaurus site.

**Reply by Chasen-Zhang** [reply]
有可能是bug吧
